src/ports/postgres/modules/deep_learning/madlib_keras_fit_multiple_model.py_in

```diff
@@ -484,7 +545,7 @@ class FitMultipleModel():
             if self.validation_table:
                 self.update_info_table(mst, False)
 
-    def run_training(self):
+    def run_training(self, mst_idx):
```

Comment:
It looks like we don't use this new parameter inside the function.

Reply:
mst_idx has been added because it helps us print per hop time when we need to debug the fit multiple query. We might also add a verbose flag in the future to print per hop time.